rust/gui-client/src-tauri/src/bin/firezone-gui-client.rs

```diff
@@ -159,6 +159,13 @@ fn run_gui(config: RunConfig) -> Result<()> {
                 return Err(anyhow);
             }
 
+            if anyhow.root_cause().is::<controller::FailedToReceiveHello>() {
+                show_error_dialog(
+                    "The Firezone Tunnel service is not responding. If the issue persists, contact your administrator.",
+                )?;
```

Comment:
@jamilbk Please make a suggestion if you think we can say something better here. There are two cases where we can (in theory) hit this:

1. We erroneously launched a 2nd instance of the GUI client when we shouldn't.
2. The Tunnel service is actually not responsive (I don't know of any reports from users where this is the case).